File: 프로그래머스/0/120837. 개미 군단/개미 군단.js
function solution(hp) {
    var answer = 0;
     if(hp%5<5){
         answer+=~~(hp/5); //ans=4
         hp=hp%5;  //hp=3
         if(hp%3<3){   
             answer+=~~(hp/3);
             hp=hp%3;  
             if(hp%1<1){
                 answer+=~~(hp/1);
             }
         }
     }
    return answer;
}

File: 프로그래머스/0/120837. 개미 군단/개미 군단.go
func solution(hp int) int {
    
    var  answer int = 0

    // 5로 나눈 몫
    answer += hp / 5
    hp = hp % 5

    // 3으로 나눈 몫
    answer += hp / 3
    hp = hp % 3

    // 1로 나눈 몫
    answer += hp / 1

    return answer
}

